src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar">
```

Comment:
modifiers is not implemented:
![image](https://github.com/mate-academy/layout_calendar/assets/102289654/eb20e532-337c-4eb0-b597-9e168e6b40a2)